README.md

```diff
@@ -26,26 +26,36 @@ In order to use them, you should rewire the default git hooks folder in your loc
 To generate the database and the go class to handle the connection run the migration tool:
 
 1. Start database and pgadmin using the command `docker-compose up -d postgres pgadmin`
-1. At the scripts/prisma folder run the `generate.sh` file using the command `sh generate.sh`
-1. Your database is started and the tables are generated
+2. At the scripts/prisma folder run the `generate.sh` file using the command `sh generate.sh`
```

Comment:
The use of "1., 1., 1.,..." for ordered lists is a good practice as if you need to insert any new item you don't have to change all the others with the respective number. Check: https://guides.github.com/features/mastering-markdown/ in te session "Ordered"

So I think we could keep the 
1.
1.
1.

Reply:
Yes, Ok